index.d.ts

```diff
@@ -1,4 +1,4 @@
-declare const enum LevelEnum {
```

Comment:
from my understanding `const enum` is a runtime optimization, and it should make no difference to the type declaration to remove it